client/components/support-info/index.jsx

```diff
@@ -17,12 +17,27 @@ function makePrivacyLink( privacyLink, link ) {
 	return null;
 }
 
+/**
+ * @typedef {import('react')} React
+ */
+
+/**
+ * Renders a support information component.
+ * @param {Object} props The component props.
+ * @param {React.ReactNode} [props.children] The child elements to be rendered inside the component.
```

Comment:
FWIW you could do it inline with `import('react').ReactNode` without the need to do a `@typedef`.

Reply:
That's so much cleaner.